src/interface/console.c

```diff
@@ -737,7 +737,17 @@ static int cc_set(const utf8 **argv, int argc)
 		}
 
 		if (strcmp(argv[0], "money") == 0 && invalidArguments(&invalidArgs, double_valid[0])) {
-			gCashEncrypted = ENCRYPT_MONEY(MONEY((int)double_val[0], ((int)(double_val[0] * 100)) % 100));
+			money32 money = MONEY((int)double_val[0], ((int)(double_val[0] * 100)) % 100);
+			if (gCashEncrypted != ENCRYPT_MONEY(money)) {
+				if (game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETMONEY, money, GAME_COMMAND_CHEAT, 0, 0) != MONEY32_UNDEFINED) {
+					//Change it locally so it shows the accurate value in the 
+					//"console_execute_silent("get money")" line when in network client mode
+					gCashEncrypted = ENCRYPT_MONEY(money);
```

Comment:
And let this thing get even more out of hand? this was something that was discussed in one of the previous pull requests and agreed upon. Unless theres already a method to install callbacks for this along with the other instances of this being done in console, i'd rather do all of that in a seperate pull request after this is merged.  it sets the value and then the cheat sets the value again so there shouldn't really be a big issue here.